osvdev/osvdev.go

```diff
@@ -0,0 +1,103 @@
+// Package osvdev provides OSV vulnerability data collection from osv.dev
+// (osv-vulnerabilities.storage.googleapis.com).
+//
+// This package implements a specific data source for osv.dev's vulnerability database,
+// which aggregates vulnerability data from multiple ecosystems including:
+// - PyPI (Python packages)
+// - Go (Go modules)
+// - crates.io (Rust crates)
+//
+// It uses the generic osv package to handle OSV format processing and provides
+// OSV.dev-specific configuration such as default URLs and directory structure.
+package osvdev
+
+import (
+	"path/filepath"
+
+	"github.com/aquasecurity/vuln-list-update/osv"
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	securityTrackerURL = "https://osv-vulnerabilities.storage.googleapis.com/%s/all.zip"
+	osvdevDir          = "osv"
+)
+
+var defaultEcosystems = map[string]osv.Ecosystem{
+	"PyPI": {
+		Dir: "python",
+		URL: "https://osv-vulnerabilities.storage.googleapis.com/PyPI/all.zip",
+	},
+	"Go": {
+		Dir: "go",
+		URL: "https://osv-vulnerabilities.storage.googleapis.com/Go/all.zip",
+	},
+	"crates.io": {
+		Dir: "rust",
+		URL: "https://osv-vulnerabilities.storage.googleapis.com/crates.io/all.zip",
+	},
+}
+
+type options struct {
+	url        string
+	dir        string
+	ecosystems map[string]osv.Ecosystem
+}
+
+type option func(*options)
+
+type Database struct {
+	osv.Database
+}
```

Comment:
It looks like we don’t need to wrap osv.Database now and can simply use osv.Database.

Reply:
Thanks. Although I did that on Seal, I forgot to make the change on osvdev.
https://github.com/aquasecurity/vuln-list-update/pull/377/commits/b776b748859ce6221b86589aa6d1333a9a23ad93